lib/db/schema.sql

```diff
@@ -18,6 +18,7 @@ CREATE TABLE IF NOT EXISTS codes (
   INDEX codes_user_id(userId),
   email VARCHAR(256) NOT NULL,
   scope VARCHAR(256) NOT NULL,
+  lastAuthAt BIGINT NOT NULL,
```

Comment:
Hmm, actually the `BIGINT` is because fxa-auth-server stores a millisecond timestamp, we probably don't need a column that big for just the seconds...